README.md

```diff
@@ -1 +1,144 @@
-# Bookstore-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+# Math Magician Project
+
+# 📗 Table of Contents
+
+- [Math Magician Project](#math-magician-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Math Magician Project ](#-math-magician-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Install ](#install-)
+    - [Usage ](#usage-)
+    - [Run tests ](#run-tests-)
+  - [👥 Authors ](#-authors-)
+    - [👤 **Mert Kantaroglu**](#-mert-kantaroglu)
+    - [👤 **Josue Rushanika**](#-josue-rushanika)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Math Magician Project <a name="about-project"></a>
+
+**Math Magician is a SPA project which you can make single calculations and reading a random math related quote**
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **Homepage**
+- **Calculator**
+- **Quote**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JS</li>
+    <li>Git</li>
+    <li>Jest</li>
+    <li>React</li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Coming soon!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Portfolio presentation will be coming soon.
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need a Chrome, Safari or Mozilla browser
+
+### Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd <desired-folder>
+  git clone https://github.com/mertkantaroglu/Math-Magician.git
+  cd math-magicians
+```
+
+### Install <a name="install"></a>
+
+Install this project with:
+
+```sh
+  npm install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```sh
+  npm run start
+```
+
+### Run tests <a name="test"></a>
+
+To run tests, run the following command:
+
+```sh
+npm run test
+```
+
+## 👥 Authors <a name="authors"></a>
+
+### 👤 **Mert Kantaroglu**
+
+- GitHub: [@mertkantaroglu](https://github.com/mertkantaroglu/)
+- Twitter: [@aranelluinwe](https://twitter.com/aranelluinwe)
+- LinkedIn: [mertkantaroglu](https://www.linkedin.com/in/mert-kantaroglu/)
+
+### 👤 **Josue Rushanika**
+
+- GitHub: [@josuerushanika](https://github.com/josuerushanika)
+- Twitter: [@josuerushanika](https://twitter.com/josuerushanika)
+- LinkedIn: [josue-rushanika](https://www.linkedin.com/in/josue-rushanika-336049b3/)
+
```

Comment:
- [ ] We only add project contributors under this section. Kindly remove this part as it is not needed.